pkg/client/query/types/blockhash.go

```diff
@@ -0,0 +1,8 @@
+package types
```

Comment:
Having separate files for this feels like overkill. 

Can we just have a `types.go`?